ASWF/meetings/tsc/2020-11-02.md

```diff
@@ -0,0 +1,203 @@
+    - Kevin: Had not made strict limitation on where boundary should lie. 
+      Without grading transforms, dynamic parameters allow you to do thing 
+      like control where an exposure change is happening. Mostly temporary 
+      adjustments. Some of the grading transforms allow me to do things where 
+      I have previously had to bake LUT, so idea of parameterized methods is 
+      useful. I'm in the middle somewhere. Think grading transform naming is 
+      wrong and misleading.
+    - Mark B: Current impl with DisplayTransform controlling where pieces are 
+      dynamic had idea of limiting (gamma, exposure) adjustments. Maybe we need 
+      more display transforms which ensure the knobs are in the proper place. 
+      Do like building block transforms to replace need for LUTs.
+    - Kevin: Yes, low level ops to support this. Isn't it up to config author 
+      to choose where to express the dynamic parameters? The way these two 
+      features come together is what creates a risky area.
+    - Doug: Yes, there is a specific tag to make things dynamic.
+    - Mark B: New building blocks could be confusing. Grading transforms don't 
+      have file to back them up.
+    - Doug: CTF format could be used.
+    - Kevin: Concern with these transforms is they are not standardized. Not a 
+      big deal having additional transformations.
+    - J: BLG is a black box, but fills a need in industry. CDL doesn't do that. 
+      We like fixed pipeline, and limited input to what can be changed. CDL has 
+      been building block for this. Who makes and communicates these decisions? 
+      How do we know what the right version for a parameter number is? Does 
+      that fit into OCIO core concepts?
+    - Mark B: Would expect to being given CTF rather than a grading transforms 
+      with dynamic parameters set.
+    - Doug: Wouldn't be handed config with dynamic parameters and expect them 
+      to edit it. Understand the concerns, but don't see how grading transforms 
+      are different from CDL in that regard.
+    - Mark B: The workflows it introduces are a big shift in the tool. If 
+      someone adjusts in their config and send the config without those 
+      numbers, there's an extra bit to account for.
+    - Kevin: If I got a config file with parameters that would be helpful. At 
+      moment we get CDLs and stuff that should match, but they don't sometimes.
+    - J: The variability of these controls is something to help facilitate 
+      re-baking LUTs. There's value in this, but our concern is this will 
+      create dynamic version of this OCIO experience that is not intended. Can 
+      forsee a future that used builtin grading tool that uses these transforms 
+      and export that to handoff to studios, but would be a nightmare. 
+    - Mark B: Or combining grading transforms in row. What will that mean?
+    - Kevin: Nothing stopping onset grading from baking out CTL files, so not 
```

Comment:
CTL --> CDL

Reply:
@KevinJW did you mean CDL here, or is CTL correct? I know we discussed both possibilities.

Reply:
CTL might work if a bit painful

CLF/CTF (not CDL), I believe I was suggesting that as an extension of CDLs there is no reason we could not have a more expressive way of passing on the on-set/dailies data (other than industry agreement !)

In this case with dynamic parameters on set, are conceptually just the same as CDL parameters on set in terms of impact, neither is sufficient you need to pass on the whole transformation chain, which could be baked to CLF/CTF. The fact that it is dynamic "on set" or in dailies is nothing new.

Reply:
Fixed.